### PR TITLE
emoacht.Monitorian version 4.2.2

### PR DESCRIPTION
**Pull request by kritzl**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/100333)

### CODE REVIEW

**Comment by kritzl** [comment]
@microsoft-github-policy-service agree

File: manifests/e/emoacht/Monitorian/4.2.2/emoacht.Monitorian.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: emoacht.Monitorian
+PackageVersion: 4.2.2
+Installers:
+- InstallerLocale: en-US
+  Architecture: x86
```
Comment (mdanish-kh) [inline manifests/e/emoacht/Monitorian/4.2.2/emoacht.Monitorian.installer.yaml:8]:
```suggestion
  Architecture: x86
  Scope: machine
```
The installation is machine-wide

File: manifests/e/emoacht/Monitorian/4.2.2/emoacht.Monitorian.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: emoacht.Monitorian
+PackageVersion: 4.2.2
+PackageLocale: en-US
+Publisher: emoacht
+PackageName: Monitorian
+License: MIT License
```
Comment (mdanish-kh) [inline manifests/e/emoacht/Monitorian/4.2.2/emoacht.Monitorian.locale.en-US.yaml:9]:
```suggestion
License: MIT License
LicenseUrl: https://github.com/emoacht/Monitorian/blob/master/LICENSE.txt
Copyright: Copyright (c) 2017 emoacht
CopyrightUrl: https://github.com/emoacht/Monitorian/blob/master/LICENSE.txt
```
Additional metadata

File: manifests/e/emoacht/Monitorian/4.2.2/emoacht.Monitorian.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: emoacht.Monitorian
+PackageVersion: 4.2.2
+PackageLocale: en-US
+Publisher: emoacht
+PackageName: Monitorian
+License: MIT License
+ShortDescription: Monitorian is a Windows desktop tool to adjust the brightness of multiple monitors with ease.
```
Comment (mdanish-kh) [inline manifests/e/emoacht/Monitorian/4.2.2/emoacht.Monitorian.locale.en-US.yaml:10]:
```suggestion
ShortDescription: Monitorian is a Windows desktop tool to adjust the brightness of multiple monitors with ease.
ReleaseNotes: |-
  - Redefine behavior of unison function #427
  - Supplement French (fr) language.
ReleaseNotesUrl: https://github.com/emoacht/Monitorian/releases/tag/4.2.2-Installer
Tags:
- monitor
- display
- brightness
- luminance
- ddc-ci
```
Additional metadata

**Review by mdanish-kh — CHANGES_REQUESTED** [review]

**Review by mdanish-kh — APPROVED** [review]

**Pull request merged**